CONTRIBUTING.md

```diff
@@ -6,8 +6,6 @@ You're always welcome to submit your PR straight away and start the discussion (
 
 ## The Akka Community
 
-If you have questions about the contribution process or discuss specific issues, please visit the [akka/dev Gitter chat](https://gitter.im/akka/dev).
-
 You may also check out these [other resources](https://akka.io/get-involved/).
```

Comment:
"also" doesn't work now, but should we have a direct link to https://discuss.akka.io/ instead of gitter?

That is include in get-involved, but might be good to have a clear replacement to gitter?